tests/e2e-complex/upgrade/common.sh

```diff
@@ -23,20 +23,19 @@ AMOUNT_BEFORE="19000000000000000"
 CHEQ_AMOUNT="1ncheq"
 CHEQ_AMOUNT_NUMBER="1"
 # shellcheck disable=SC2034
-DID_1="did:cheqd:testnet:abcdef"
+DID_1="did:cheqd:testnet:1111111111111111"
 # shellcheck disable=SC2034
-DID_2="did:cheqd:testnet:higklm"
+DID_2="did:cheqd:testnet:2222222222222222"
 
 # cheqd_noded docker wrapper
-
 cheqd_noded_docker() {
-  docker run --rm \
-    -v "$(pwd):/cheqd" \
-    --network host \
-    -u root \
-    -e HOME=/cheqd \
-    --entrypoint "cheqd-noded" \
-    ${CHEQD_IMAGE_FROM} "$@"
+    docker run --rm \
+        -v "$(pwd):/cheqd" \
+        --network host \
+        -u root \
+        -e HOME=/cheqd \
+        --entrypoint "cheqd-noded" \
+        ${CHEQD_IMAGE_FROM} "$@"
```

Comment:
Shouldn't we use a similar Docker compose up technique like the pipeline? + not a good idea to test as root (cheqd user should be default in the build process, so we could change this to cheqd)

Reply:
It's the CLI image. I suggest changing this to direct cheqd-cli calls in future. cc @anikitinDSR. And yes, probably we could use the latest docker-compose as well.

Reply:
I'm going to create a follow-up ticket and merge in order to make a release and start testing. This test is isolated so we can improve it later.

Reply:
Ticket: https://app.clickup.com/t/28t9wfg